scripts/install-assets.sh

```diff
@@ -14,5 +14,5 @@ echo "Installing Node dependencies..."
 npm install --silent
 
 echo -n "Building Node JS/CSS assets..."
-npm run production > /dev/null 2>&1
+npm run production
```

Comment:
Curious if we know why this was sent to /dev/null? Any negative consequences of removing that?

Reply:
Thanks! I removed it to see the deployment output on forge but will keep it.